src/main/java/org/toradocu/extractor/JavadocExtractor.java

```diff
@@ -305,10 +340,16 @@ private ParamTag createParamTag(JavadocBlockTag blockTag, List<DocumentedParamet
 
     final List<DocumentedParameter> matchingParams =
         parameters.stream().filter(p -> p.getName().equals(paramName)).collect(toList());
-    // TODO If paramName not present in paramNames => issue a warning about incorrect documentation!
-    // TODO If more than one matching parameter found => issue a warning about incorrect documentation!
-    Comment commentObject = new Comment(blockTag.getContent().toText());
-    return new ParamTag(matchingParams.get(0), commentObject);
+
+    if (matchingParams.isEmpty()) {
+      throw new ParameterNotFoundException(
+          "Documented parameter " + paramName + " does not correspond to any formal parameter.");
+    } else if (matchingParams.size() > 1) {
+      throw new AssertionError("Error: multiple formal parameters with the same name.");
```

Comment:
In this case we always want to throw exception without checking the `--stop-on-error` value?